rocketpy/mathutils/function.py

```diff
@@ -3261,12 +3353,19 @@ def __validate_extrapolation(self, extrapolation):
 
         ## multiple dimensions
         elif self.__dom_dim__ > 1:
-            if extrapolation not in [None, "natural"]:
+            if extrapolation is None:
+                extrapolation = "natural"
+            if extrapolation == "natural" and self.__interpolation__ == "linear":
+                warnings.warn(
+                    "Extrapolation 'natural' is not supported for multidimensional "
+                    "linear interpolation. 'rbf' will be used to extrapolate."
+                )
+            elif extrapolation not in ["constant", "natural", "zero"]:
                 warnings.warn(
                     "Extrapolation method set to 'natural'. Other methods "
```

Comment:
this warning message needs to be updatred

Reply:
Solved by [86458aa](https://github.com/RocketPy-Team/RocketPy/pull/691/commits/86458aab47331eaeb59f9c61795f84548c5eb90c)